protocol/migrations/sqlite/1712447150_update_raw_messages_with_resend_features.up.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE raw_messages ADD COLUMN sender BLOB;
```

Comment:
Question: would it make sense to store resend-related columns in a separate table? So that, for example, maintain a resend history for each raw message for some statistics. And also in order to separate the actual message data (payload etc.) from resend-related data.